auth-api/src/auth_api/resources/user.py

```diff
@@ -218,3 +218,33 @@ def get():
         """Add a new contact for the Entity identified by the provided id."""
         sub = g.jwt_oidc_token_info.get('sub', None)
         return AuthorizationService.get_user_authorizations(sub), http_status.HTTP_200_OK
+
+
+@cors_preflight('PATCH, OPTIONS')
+@API.route('/termsofuse', methods=['PATCH', 'OPTIONS'])
+class UserTermsOfUse(Resource):
+    """Resource for managing users terms of use."""
+
+    @staticmethod
+    @TRACER.trace()
+    @cors.crossdomain(origin='*')
+    @_JWT.requires_auth
+    def patch():
+        """Update terms of service for the user."""
+        token = g.jwt_oidc_token_info
+        request_json = request.get_json()
+
+        valid_format, errors = schema_utils.validate(request_json, 'termsofuse')
+        if not token:
```

Comment:
This token check is not needed, but I think we have techdebt ticket remove these unwanted token checks